### ISSUE TITLE
this.loadArgMetadata is not a function (npm 3.1.1)

### ISSUE THREAD

**Issue by substack** [npm/npm#8911]
I was getting this error in 3.0.0 so I upgraded to 3.1.1 but am still getting it:

```
$ npm install --save through2
npm ERR! Linux 3.16.0-4-amd64
npm ERR! argv "/home/substack/prefix/bin/node-iojs" "/home/substack/prefix/bin/npm" "install" "--save" "through2"
npm ERR! node v2.0.0
npm ERR! npm  v3.1.1

npm ERR! this.loadArgMetadata is not a function
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /home/substack/projects/wherewebreathe/npm-debug.log
```

and the log:

```
$ cat npm-debug.log 
0 info it worked if it ends with ok
1 verbose cli [ '/home/substack/prefix/bin/node-iojs',
1 verbose cli   '/home/substack/prefix/bin/npm',
1 verbose cli   'install',
1 verbose cli   '--save',
1 verbose cli   'through2' ]
2 info using npm@3.1.1
3 info using node@v2.0.0
4 silly loadCurrentTree Starting
5 silly install loadCurrentTree
6 silly install readLocalPackageData
7 verbose stack TypeError: this.loadArgMetadata is not a function
7 verbose stack     at /home/substack/projects/npm/lib/install.js:585:21
7 verbose stack     at /home/substack/projects/npm/node_modules/graceful-fs/graceful-fs.js:76:16
7 verbose stack     at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:382:3)
8 verbose cwd /home/substack/projects/wherewebreathe
9 error Linux 3.16.0-4-amd64
10 error argv "/home/substack/prefix/bin/node-iojs" "/home/substack/prefix/bin/npm" "install" "--save" "through2"
11 error node v2.0.0
12 error npm  v3.1.1
13 error this.loadArgMetadata is not a function
14 error If you need help, you may report this error at:
14 error     <https://github.com/npm/npm/issues>
15 verbose exit [ 1, true ]
```

**Comment by iarna** [comment]
I don't see how this could show up in `3.0.0` or `3.1.0` as `loadArgMetadata` and code calling it wasn't added until `3.1.1`, but regardless, this will be fixed by #8922.

**Comment by iarna** [comment]
This was fixed by 3.1.2
